docs/index.rst

```diff
@@ -2,54 +2,84 @@
 sunpy core Documentation
 ************************
 
+.. image:: tour.svg
+  :align: center
+
 ``sunpy`` is a community-developed, free and open-source solar data analysis environment for Python.
 It provides search and download functionality, data containers for image and time series data, as well as commonly used coordinate frames and transformations between such frames.
```

Comment:
```suggestion
``sunpy`` among other things provides search and download functionality, data containers for image and time series data, as well as commonly used coordinate frames and transformations between such frames.
```
I think this is what Stuart wanted. Tho my corrections are bad. The sentence suggests we only do those three things.

Reply:
How about just changing "provides" to "includes"? That perhaps is a bit less exclusionary. Also, is there any major feature of the package we are missing with this description?

Reply:
I took another stab at it. Let me know what you think (and maybe this makes @Cadair a bit happier too)